proposal.rst

```diff
@@ -168,82 +173,124 @@ Project Details
 
 **Describe any existing work and concepts on which your project is based.**
 
-FILLME
+First, I have collected several resources about how to implement and improve the estimation.
+
+*** Academic
+1. (Estimating Corpus Size via Queries [CIKM'06])[http://fontoura.org/papers/cikm2006.pdf] This paper uses the basic estimator estimate the search result size. The first approach requires a uniform random sample of documents from the corpus. It avoids this notoriously difficult sample generation problem, and instead uses two fairly uncorrelated sets of terms as query pools; the accuracy of the second approach depends on the degree of correlation among the two sets of terms.
```

Comment:
This paper is describing how to estimate the total number of documents in the collection when you can only run queries and see how many documents match (for example, you could use this to estimate how many pages Google index).

That's a related problem to the one we want to solve, but it seems to me to be essentially a different problem.  We already exactly know the collection size, but when running a query we want to try to return a better estimate of the number of matches.  If we would need to run extra queries to improve the estimate we might as well just consider additional documents for the query we're already running.

Reply:
Got it.